packages/in_app_purchase/in_app_purchase_ios/lib/src/store_kit_wrappers/sk_payment_queue_wrapper.dart

```diff
@@ -473,3 +482,99 @@ class SKPaymentWrapper {
   @override
   String toString() => _$SKPaymentWrapperToJson(this).toString();
 }
+
+/// Dart wrapper around StoreKit's
+/// [SKPaymentDiscount](https://developer.apple.com/documentation/storekit/skpaymentdiscount?language=objc).
+///
+/// Used to indicate a discount is applicable to a payment. The
+/// [SKPaymentDiscountWrapper] instance should be assigned to the
+/// [SKPaymentWrapper] object to which the discount should be applied.
+/// Discount offers are set up in App Store Connect.
```

Comment:
Maybe offer a link for how to set up the discount offer? (if any)

Reply:
Done.